test/README.md

```diff
@@ -1,12 +1,21 @@
-The tests in the `./tests/` directory were originally copied from the [messageformat project](https://github.com/messageformat/messageformat/tree/11c95dab2b25db8454e49ff4daadb817e1d5b770/packages/mf2-messageformat/src/__fixtures)
-and are here relicensed by their original author (Eemeli Aro) under the Unicode License.
+# Unicode MessageFormat Test Suite
 
-These test files are intended to be useful for testing multiple different message processors in different ways:
+These test files are intended to be useful for testing multiple different _message_ processors in different ways:
 
 - `syntax.json` — Test cases that do not depend on any registry definitions.
 
 - `syntax-errors.json` — Strings that should produce a Syntax Error when parsed.
 
+> [!NOTE]
+> `syntax-errors.json` does not include tests for the disallowed uses of unpaired surrogate code points
+> because JSON does not permit unpaired surrogate code points.
+> If your implementation uses UTF-16 based strings (such as JavaScript `String` or Java `java.lang.String`)
+> or otherwise allows unpaired surrogates in text or literals, you will need to implement tests equivalent
+> to the following for syntax errors:
+> ```json
+> { "src": "{\ud800}" }
```

Comment:
This should be a fully qualified test, rather than relying on the `syntax-errors.json` `"defaultTestProperties"`.
```suggestion
> {
>   "locale": "en-US",
>   "src": "{\ud800}",
>   "expErrors": [{ "type": "syntax-error" }]
> }
```